content/posts/newsletter/2020-09-06.md

```diff
@@ -0,0 +1,134 @@
+
+### Clients and applications
+
+A new major version of Movim is out. With improved notifications, new video-conferencing and chat features and some major performance overhaul. A new version for the Android application is also available to integrate all those features in the Android environment. Discover all the improvements of this new release on the official release note [Movim 0.18 – Oterma](https://nl.movim.eu/?node/pubsub.movim.eu/Movim/11655111-e7ad-4e0c-975c-3c78755d22aa). Have a look at Movim’s improved draw widget:
+
+![Movim](/images/newsletter/september2020/movim.png "Movim")
+
+[Gajim Development News (September)](https://gajim.org/post/2020-09-27-development-news-september/): This month’s post marks one year of Gajim Development News! The rework of Gajim’s Preferences window has been completed. We improved group chat invitations and made group chat settings easier to access. Also, joining a group chat is now much prettier.
+
+MCabber, the venerable old text UI client, has been released in [version 1.1.1](https://github.com/McKael/mcabber/releases/tag/1.1.1) and [1.1.2](https://github.com/McKael/mcabber/releases/tag/1.1.2). Apart from bugfixes, support for displaying the number of unread messages and refactorization of carbon message handling has been implemented.
+
+The iOS and MacOS client Monal got a [new push server](https://monal.im/blog/monal-push-server-upgrade/) and received improvements for Message Archive Management. This is one of the many changes to arrive in future releases. Be prepared and get a glimpse in the [Monal Beta Testflight](https://monal.im/blog/news-betas-up-for-4-8/).
+
+The Python SDK for GE WiFi-enabled kitchen appliances, [gekitchen](https://github.com/ajmarks/gekitchen), made its first GitHub release. The primary goal is to use it to power integrations for [Home Assistant](https://www.home-assistant.io/), though that probably needs to wait for some new entity types. It is using XMPP as a mean to authenticate, since it is what the The GE SmartHQ app uses to communicate with devices.
+
+Hello [BeagleIM 4.0 and SiskinIM 6.0](https://tigase.net/beagleim-4.0-and-siskin-6.0-released/)! Their iOS and MacOS clients now speak Mediated Information eXchange (MIX) [(XEP-0369, XMPP’s new group chat protocol)](https://xmpp.org/extensions/xep-0369.html). VoIP connectivity has been improved with Jingle Message Initiation (XEP-0353), Last Message Correction ([XEP-0308](https://xmpp.org/extensions/xep-0308.html)) is available for editing messages which were already sent, and other features like Message Retraction ([XEP-0424](https://xmpp.org/extensions/xep-0424.html)) and quick replies for quoting were introduced. Support for CAPTCHA forms has been added as well: Extended In-Band Registration ([XEP-0158](https://xmpp.org/extensions/xep-0158.html)) is invoked for additional verification during account registration. Of course there are many more changes and bugfixes! XMPP on iOS and Mac is getting traction after all!
```

Comment:
I would switch Monal's section with gekitchen's  and write "XMPP on iOS and Mac is getting traction after all!" 
in a new paragraph, so that sentence can be shared for both iOS/Mac projects